doc/changelog.md

```diff
@@ -312,22 +464,24 @@
   register.
   [(#784)](https://github.com/PennyLaneAI/catalyst/pull/784)
 
-* Correctly querying batching rules for `jax.scipy.linalg.expm`
+* `jax.scipy.linalg.expm` is now supported within qjit-compiled functions.
   [(#733)](https://github.com/PennyLaneAI/catalyst/pull/733)
+  [(#752)](https://github.com/PennyLaneAI/catalyst/pull/752)
 
-* Correctly linking openblas routines necessary for `jax.scipy.linalg.expm`.
+  This required correctly linking openblas routines necessary for `jax.scipy.linalg.expm`.
   In this bug fix, four openblas routines were newly linked and are now discoverable by
   `stablehlo.custom_call@<blas_routine>`. They are `blas_dtrsm`, `blas_ztrsm`, `lapack_dgetrf`,
   `lapack_zgetrf`.
-  [(#752)](https://github.com/PennyLaneAI/catalyst/pull/752)
 
-* Correctly recording types of constant array when lowering `catalyst.grad` to mlir
+* Fixes a bug where QNodes that contained `QubitUnitary` with a complex matrix
+  would error during gradient computation.
   [(#778)](https://github.com/PennyLaneAI/catalyst/pull/778)
 
 * Callbacks can now return types which can be flattened and unflattened.
   [(#812)](https://github.com/PennyLaneAI/catalyst/pull/812)
 
-* `catalyst.qjit` and `catalyst.grad` can now get `__name__` from `functools.partial`.
+* `catalyst.qjit` and `catalyst.grad` now work correctly with
+  functions that have been wrapped with `functools.partial`.
   [(#820)](https://github.com/PennyLaneAI/catalyst/pull/820)
 
 <h3>Internal changes</h3>
```

Comment:
771 and 768 need to be updated. Instead of generating a new class `Controlled`/`Adjoint` that inherits from both catalyst and pennylane classes, using adjoint/ctrl on a single operation will now directly dispatch to the pennylane class.

Reply:
739 should have a different description or be removed entirely. I don't think that linting needs to appear in the change log.

Reply:
We might want to consider moving 711 up out of the internal changes. It is somewhat useful to know that the operation object can now be accessed from the control flow decorators, especially for developers writing transforms.

Reply:
Program verification 626/783 might even be considered a new feature no? Definitely much too user-facing for an internal change. It also deserves a better description than just one line.

Reply:
- Have updated 771 and 768, and also added in a link to 802. Note that 771/768/802 are also listed in the Improvements section, so we could potentially remove their listings in Internal Changes.

- 739 has been removed

- I've moved 711 to improvements. Something that comes to mind -- is this documented anywhere outside of the changelog? It might get lost from awareness otherwise

- 626/783: I've moved this to the top of improvements, and added a code example

Reply:
>I've moved 711 to improvements. Something that comes to mind -- is this documented anywhere outside of the changelog? It might get lost from awareness otherwise

Good point. It should probably be included in the for_loop/while_loop/cond docstring if it isn't already!